index.html

```diff
@@ -108,6 +108,7 @@
 
       openmct.install(openmct.plugins.Espresso());
       openmct.install(openmct.plugins.MyItems());
+      openmct.install(openmct.plugins.ActivityStates());
```

Comment:
Does it make sense to have this be an independent plugin if it's dependent on Plans?

Reply:
Good point!
I've removed the plugin and added the interceptor to the PlanLayout plugin.